src/RecoveryServices/RecoveryServices/Vault/UpdateAzureRmRecoveryServicesVault.cs

```diff
@@ -77,9 +77,21 @@ public class UpdateAzureRmRecoveryServicesVault : RecoveryServicesCmdletBase
         [ValidateNotNullOrEmpty]
         public SwitchParameter RemoveSystemAssigned { get; set; }
 
+        /// <summary>
+        /// Enables or disables Classic alerts for RS vault.
+        /// </summary>
+        [Parameter(Mandatory = false)]        
+        public bool? DisableClassicAlerts { get; set; }
```

Comment:
use singular parameter name
```suggestion
        public bool? DisableClassicAlert { get; set; }
```

Reply:
Alerts here refer to Alerts feature settings and that's how it would be more meaningful in this scenario. Pls let me know if I still need to change it.